tests/parametric/conftest.py

```diff
@@ -398,35 +399,47 @@ def pytest_runtest_makereport(item, call):
 
 @pytest.fixture
 def test_server_log_file(apm_test_server, request) -> Generator[TextIO, None, None]:
-    with tempfile.NamedTemporaryFile(mode="w+") as f:
+
+    log_path = f"{context.scenario.host_log_folder}/outputs/{request.cls.__name__}/{request.node.name}/server_log.log"
+    os.makedirs(os.path.dirname(log_path), exist_ok=True)
+    with open(log_path, "w") as f:
         yield f
-        f.seek(0)
-        request.node._report_sections.append(
-            ("teardown", f"{apm_test_server.lang.capitalize()} Library Output", "".join(f.readlines()))
```

Comment:
I personally find the pytest report sections very useful for debugging as it puts all the debugging info right next to the test case. Can we keep this?

Reply:
But after the integration on "system-tests scenario", we are removing this section here: https://github.com/DataDog/system-tests/blob/ca9a5dad20070144e64cdeb2904541e478b063d2/conftest.py#L267

Reply:
This line removes those section in reports.json only no? Did I missed something ?